src/main/java/vazkii/patchouli/common/network/message/MessageOpenBookGui.java

```diff
@@ -14,12 +14,9 @@
 
 import java.util.function.Supplier;
 
-public class MessageOpenBookGui {
-	public static final ResourceLocation ID = new ResourceLocation(Patchouli.MOD_ID, "open_book");
+public record MessageOpenBookGui(ResourceLocation book, @Nullable ResourceLocation entry, int page) {
```

Comment:
whats the point? The fields are now publicly accessible when they weren't before